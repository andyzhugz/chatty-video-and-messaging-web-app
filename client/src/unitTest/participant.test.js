/* eslint-disable react/jsx-filename-extension */
/* eslint-disable no-undef */
import React from 'react';
import ReactDOM from 'react-dom';
import { act, Simulate } from 'react-dom/test-utils';
import { BrowserRouter as Router } from 'react-router-dom';
import axios from 'axios';

import Participant  from '../components/Participant';
const renderer = require('react-test-renderer');

jest.mock('axios');

// snapshot testings
describe('Test snapshot', () => {
    test('Test Participant', () => {
      const component = renderer.create(<Router><Participant participant={{
        params: {
          param1: 'test',
        }
      }}/></Router>);
      const tree = component.toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
