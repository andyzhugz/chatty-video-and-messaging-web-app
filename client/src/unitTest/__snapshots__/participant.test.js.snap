// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test snapshot Test Participant 1`] = `
<div
  className="participant"
>
  <h3 />
  <video
    autoPlay={true}
  />
  <audio
    autoPlay={true}
  />
</div>
`;
